source/api/commands/as.md

```diff
@@ -70,6 +91,47 @@ cy.wait('@userPut')
 
 Some strings are not allowed as alias names since they are reserved words in Cypress. These words include: `test`, `runnable`, `timeout`, `slow`, `skip`, and `inspect`.
 
+## `as` is asynchronous
+
+Remember that **Cypress commands are async**, including `as()`.
+
+Because of this you cannot _synchronously_ access anything you have aliased. You must use other asynchronous commands such as {% url `.then()` then %} to access what you've aliased.
+
+Here are some further examples of using `as()` that illustrate the asynchronous behaviour.
```

Comment:
behavior -> behavior - I know I need to write up our official style guide, but we follow US english spellings